pandas-stubs/core/frame.pyi

```diff
@@ -94,13 +97,18 @@ class DataFrame:
     #
     # methods
     def append(
-        self, s: Union[DataFrame, Dict[_str, Any]], ignore_index: bool = ..., sort: bool = ...,
+        self, s: Union[DataFrame, Dict[_str, Any]], ignore_index: bool = ..., sort: bool = ...
     ) -> DataFrame: ...
     @overload
     def apply(self, f: Callable[..., int]) -> Series: ...
     @overload
     def apply(self, f: Callable[..., _ListLike]) -> DataFrame: ...
-    def astype(self, dtype: _str) -> DataFrame: ...
+    def astype(
+        self,
+        dtype: Union[_str, Type[_np.dtype], Dict[_str, Type[_np.dtype]]],
+        copy: bool = True,
+        errors: _ErrorType = 'raise',
```

Comment:
We follow the recommendation of not putting the default values into the type stubs.

```suggestion
        copy: bool = ...,
        errors: _ErrorType = ...,
```